lib/ldap_fluff/posix_member_service.rb

```diff
@@ -21,7 +21,7 @@ def find_user_groups(uid)
     @ldap.search(
       :filter => user_group_filter(uid, user[:dn].first),
       :base => @group_base, :attributes => ["cn"]
-    ).map { |entry| entry[:cn][0] }
+    )&.map { |entry| entry[:cn][0] }
```

Comment:
Looking at [the change](https://github.com/theforeman/ldap_fluff/commit/3d11c47169ab89436a025f7bd1728fcc868feab2) that introduced `map` here, it seems the original intent was to always return an array, although it was still susceptible to crashing hard if `@ldap.search` returned `nil` even before the map change.

```suggestion
    )&.map { |entry| entry[:cn][0] } || []
```